internal/pkg/providers/aws/securityGroupUpdator.go

```diff
@@ -0,0 +1,98 @@
+package aws
```

Comment:
Perhaps merge with the class above? I think it would be much better to have relevant code in one class.